pynxtools/dataconverter/convert.py

```diff
@@ -75,7 +75,8 @@ def convert(input_file: Tuple[str],
     # Reading in the NXDL and generating a template
     definitions_path = nexus.get_nexus_definitions_path()
     if nxdl == "NXtest":
-        nxdl_path = os.path.join("tests", "data", "dataconverter", "NXtest.nxdl.xml")
+        nxdl_path = os.path.join(f"{definitions_path}/../../", "tests", "data",
```

Comment:
rather use the module path, because the locations of the nexus definition can be overridden by environment variable

Reply:
Handled in by the following new commit 45f257

Reply:
https://stackoverflow.com/questions/247770/how-to-retrieve-a-modules-path